### DISCUSSION TITLE
Welcome to PHP-Floorplan-and-advanced-automation-for-Domoticz Discussions!

### DISCUSSION THREAD

**Discussion by Egregius** [Egregius/PHP-Floorplan-and-advanced-automation-for-Domoticz#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sincze** [comment]
**Who Am I**
A Domoticz enthousiast with a fully automated home. I started working with Pass2PHP as my Domoticz reaction time was getting slow. I iterated since the beginning over the several Pass2PHP versions that were released and even contributed some functions. I am currently running a stable "fork", not utilizing the great Floorplans at the moment on a Raspberry Pi 4 / 8 GB. 

**Mission**
I try to help others setup and maintain their setup to achieve the maximum performance with Domoticz and to have them and their families enjoy Domoticz at its best..

In addition I release scripts based on the Pass2PHP philosophy on my GitHub page for others to use as well.
